managers/views.py

```diff
@@ -69,6 +70,12 @@ class ProjectDetailView(DetailView):
     template_name = "managers/project_detail.html"
     model = Project
 
+    def get_context_data(self, **kwargs: Any) -> dict:
+        context = super().get_context_data(**kwargs)
+        context["year"] = timezone.now().year
```

Comment:
maybe a silly question, but why do we use `timezone.now()` here when in (almost?) all other cases we use `datetime.now()`?

Reply:
Generally, I treat them interchangeably, since timezones are not a big deal for the most part. There was one instance when `timezone` had to be used over `datetime` in unit tests, because of issues with `freezegun` when comparing timestamps. Majority of files I've been working on were already using datetime, so I just stuck to it. Here, there was no such module before, so I picked `timezone`. I guess I just consider it more reliable. I also like the simpler syntax, instead of awkward double **datetime**.

Yeah, I guess I could just import `date` and `datetime` separately and refactor the existing code to avoid it...
Or perhaps, if we were to change the use of `datetime` in code, we should drop it altogether in favour of Django's `timezone`?

Reply:
maybe we could refactor out code to use `django`'s `timezone`, but I think it should be another issues.
@rwrzesien what do you think?

Reply:
@dybi Yes, there was an issue for that I think to change all usages to `timezone`. Basically it does the same as datetime, but takes into consideration Django timezone settings (defined in settings file). So if at some point of the project we decide to use timezone(s) or change current timezone it is already done everywhere already. So I would say its just the matter of good Django practice.